CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ## 12.0.1 under development
 
-- no changes in this release.
+- Enh #282: Allow to use the name of the view starting with `../` to join the parent directory containing the view 
+  currently being rendered (@vjik)
```

Comment:
What's the use-case for it?

Reply:
When view placed in parent directory, e. g. `../_header.php`.

Reply:
What if it's two levels up?

Reply:
It's work. I just added such test.

Reply:
While it might be handy, it poses a potential security issue if view name is obtained from user's input, i.e. user may use `../../etc/passwd` or something alike to read files.

Reply:
Seems, it's was always available. For example, `./../../etc/passed/`.

Should we somehow prevent this (I don't have ideas how)? Or note adding to render() method will be enough?

Reply:
A note should be enough, I think.

Reply:
Done